### DISCUSSION TITLE
Happy Easter!!!

### DISCUSSION THREAD

**Discussion by madmartin** [tbnobody/OpenDTU#785] [General]
![Screenshot_20230408_214500](https://user-images.githubusercontent.com/4208120/230740147-90d7152d-3c9e-4566-a897-4012fa8e4324.png)

**Comment by Solarteur** [comment]
The easter bunny was with me too!

**Comment by 3DJupp** [comment]
Nice, saw that aswell 🥚 
![image](https://github.com/tbnobody/OpenDTU/assets/8407566/680af4a0-abd2-46f2-bcb9-08a48658e18c)
